web/private/.gitignore

```diff
@@ -0,0 +1,6 @@
+# Ignore everything in this directory
+*
+# Except
+!.gitignore
+!config.default.ini
+!InitMySQL.sql
```

Comment:
Why do you want to expose the initialization script of MySQL database from the site?

Reply:
That private folder should not be reachable from the web, I've missed the .htaccess in the root web folder.
Inside private folder there is the mysql configuration file and sqlite DB it's created there, so it's should be all unreachable (but not gitignored because of sample mysql config and initialization script)